### DISCUSSION TITLE
How can I communicate with the nodes in whitefield outside the simulation?

### DISCUSSION THREAD

**Discussion by chillaxdev** [whitefield-framework/whitefield#123] [Q&A]
Node 1 Logs

> [INFO: whitefield] set gNodeID = 1
> INFO  00:00:00.000 [init:191] initing wfradio
> INFO  00:00:00.000 [usock_init:67] USOCK binding to [/WHITEFIELD_0_00010001]
> INFO  00:00:00.000 [init:125] Initing wfmac_driver
> [INFO: Main      ] Starting Contiki-NG-develop/v4.5-354-g835268ce6
> [INFO: Main      ] - Routing: RPL Lite
> [INFO: Main      ] - Net: sicslowpan
> [INFO: Main      ] - MAC: wfmac
> [INFO: Main      ] - 802.15.4 PANID: 0xabcd
> [INFO: Main      ] - 802.15.4 Default channel: 26
> [INFO: Main      ] Node ID: 1
> [INFO: Main      ] Link-layer address: 0102.0304.0506.0001
> [INFO: RPL       ] initializing
> [INFO: Main      ] Tentative link-local IPv6 address: fe80::302:304:506:1
> [INFO: whitefield] Added global IPv6 address fd00::302:304:506:1
> [INFO: whitefield] MAC wfmac NETWORK sicslowpan
> [INFO: whitefield] route_table_max_sz=300,nbr_table_max_sz=300
> [INFO: App       ] Starting Erbium Example Server

I compiled the coap-server example on contiki-ng and ran it on node 1 in Whitefield Simulation.

The global ipv6 of node 1 is fd00::302:304:506:1 from the logs. I tried pinging, it says "Network Unreachable". I tried a GET request using a COAP client (libcoap2 to be specific). However, it says the same error. Am I missing something? 

Any help is appreciated. Thank You.

PS: I assumed the problem could be not having a border router. So I tried to compile rpl-border-router and deploy it on node 0. However, make command fails because there's no implementation for slip-arch.c for Whitefield (slip_arch_writeb & slip_arch_init undefined reference errors). 

Regards
Shiv

**Comment by nyrahul** [comment]
Yes, the problem is the border router. In general, the current border router does not connect to the fallback network today using slip. This needs to be handled.  I would be happy to work on this along with you if there's interest.
